packages/package.json

```diff
@@ -32,6 +32,7 @@
     "d3-path": "^2.0.0",
     "d3-scale": "^4.0.2",
     "date-fns": "^2.21.1",
+    "internmap": "^2.0.3",
```

Comment:
what's the reason for declaring this now? It looks like we're moving it around in our build step, not actually using it. Is it a nested dependency